autorest/adal/cli_token.go

```diff
@@ -0,0 +1,32 @@
+package adal
+
+import "strconv"
+
+type AzureCLIToken struct {
+	AccessToken      string `json:"accessToken"`
+	Authority        string `json:"_authority"`
+	ClientID         string `json:"_clientId"`
+	ExpiresIn        int     `json:"expiresIn"`
+	ExpiresOn        string `json:"expiresOn"`
+	IdentityProvider string `json:"identityProvider"`
+	IsMRRT           bool    `json:"isMRRT"`
+	RefreshToken     string `json:"refreshToken"`
+	Resource         string `json:"resource"`
+	TokenType        string `json:"tokenType"`
+	UserId           string `json:"userId"`
+}
+
+func AzureCLIAccessTokensPath() string {
+	return "~/.azure/accessTokens.json"
```

Comment:
I don't believe this is a portable way of finding the Azure CLI directory. When I run [this program](https://play.golang.org/p/H_YnVgs8uP) on my Windows machine I get unhelpful results.

Mind tinkering with this to find a more portable method?